css/style.css

```diff
@@ -0,0 +1,109 @@
+/***************************/
+/* HEADER */
+/**************************/
+.header {
+  background-image: url(/img/Bg.png);
+  box-shadow: inset 0px 70px 40px rgb(0 0 0 / 30%);
+}
+
+.logo {
+  color: #fff;
+  font-family: "Yeseva One", cursive;
+  font-size: 3.6rem;
+  font-weight: lighter;
```

Comment:
I'd suggest to use numeric values. Like `font-weight: 400;`
It's more universal